extensions/stackdriver/stackdriver.h

```diff
@@ -105,7 +108,10 @@ class StackdriverRootContext : public RootContext {
 
   long int last_edge_report_call_nanos_ = 0;
 
-  long int edge_report_duration_nanos_;
+  long int edge_new_report_duration_nanos_;
```

Comment:
nit: initialize it with the 1m? Even it will definitely be set during on onConfigure, but it would still be nice to give it a default value to prevent random behavior because of future change.

Reply:
done.